hack/deployer/runner/settings.go

```diff
@@ -26,15 +26,16 @@ type Plan struct {
 	Gke               *GkeSettings `yaml:"gke,omitempty"`
 	Aks               *AksSettings `yaml:"aks,omitempty"`
 	Ocp               *OcpSettings `yaml:"ocp,omitempty"`
+	EKS               *EKSSettings `yaml:"eks,omitempty"`
```

Comment:
Abbreviations should be all caps afaik. I think we should refactor that, but maybe in a separate PR?

Reply:
Yes, we should. I have nothing against doing it in this PR :)

Reply:
Let's keep things separate https://github.com/elastic/cloud-on-k8s/issues/2815